Makefile

```diff
@@ -76,3 +76,24 @@ $(BUILD_DIR)/gofumpt: $(BUILD_DIR)
 
 $(BUILD_DIR):
 	@mkdir -p $(BUILD_DIR)
+
+.PHONY: install-jq
+install-jq: ## Install jq if not available
```

Comment:
We'll need to do this in the CI configs if we don't use the make file target here.

Reply:
Taking care of that here - https://github.com/openshift/release/pull/69497